ui/site/src/main.js

```diff
@@ -612,9 +612,10 @@ lichess.topMenuIntent = function() {
         return false;
       });
 
-      $('#ham-plate').one('mouseover click', function() {
-        $('body').append($('<div id=fpmenu>').load('/fpmenu'));
+      $('#ham-plate').one('mouseover', function() {
```

Comment:
touchscreen don't have a mouseover event. Try the chrome dev tools device emulator.